bessctl/cli.py

```diff
@@ -441,6 +430,52 @@ def restore_echoctl(self):
         except:
             pass
 
+    def go_interactive(self):
```

Comment:
It was long enough ago that my memory is foggy, but IIRC, my earlier version was "maybe_go_interactive" because there was at least one case where it didn't: when you already *were* in interactive mode.  The idea being that actually typing "interactive" when you're already in interactive mode should do nothing rather than actually try to go into interactive mode again.